src/main/scala/rocket/RocketCore.scala

```diff
@@ -876,12 +876,12 @@ class Rocket(tile: RocketTile)(implicit p: Parameters) extends CoreModule()(p)
   coreMonitorBundle.valid := csr.io.trace(0).valid && !csr.io.trace(0).exception
   coreMonitorBundle.pc := csr.io.trace(0).iaddr(vaddrBitsExtended-1, 0).sextTo(xLen)
   coreMonitorBundle.wren := wb_wen && !wb_set_sboard
-  coreMonitorBundle.wrdst := wb_waddr
+  coreMonitorBundle.wrdst := Mux(wb_ctrl.wxd || wb_ctrl.wfd, wb_waddr, 0.U)
```

Comment:
I still don't agree with this change.  0 is ineffective as a sentinel value because it's a valid FP register.  Regardless.  we have the `wren` signal above.  This signal is a don't-care when `wren` is false.

Reply:
Agree that this change can't and doesn't change function.
Would it be better if I changed the log printout values, instead of here in the coreMonitorBundle?
If we change the printout, then we can use other values than 0, like spaces or hyphens.
Or should we just leave it be and not complicate too much?
Just trying to make the simulation log prettier to read, not changing function.

Reply:
I think making the printout easier to read is a worthy goal, but I do think those improvements are best isolated to the printing code, so that it’s clear we aren’t trying to add extra semantics or constraints to the monitoring logic.

You might want to shop around the various options for improving the printout. Moving these muxes over there would help right now and isn’t controversial, but other changes that improve the visuals could upset code that parses the printout.